shared/components/ColoredTextProvider/ColoredTextProvider.jsx

```diff
@@ -9,21 +9,21 @@ class ColoredTextProvider extends React.Component {
   }
 
   render() {
-    const { colorClassName, children } = this.props
-    if (colorClassName === null) {
+    const { className, children } = this.props
```

Comment:
This is just a question. I'm curious what was the rationale for changing `colorClassName` to `className`?
This will be a breaking change.